client/packages/system/src/Item/Components/ItemVariant/PackUnitCell.tsx

```diff
@@ -11,12 +11,17 @@ export const getPackUnitCell =
   <T extends RecordWithId>({
     getItemId,
     getPackSize,
+    getUnitName,
   }: {
     getItemId: (row: T) => string;
     getPackSize: (row: T) => number;
+    getUnitName: (row: T) => string | null;
```

Comment:
decide to use null rather then undefined, because if it's undefined it's easily missed in useUnitVariant